src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/ManifestAssemblyMvidHeaderNode.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+
+using Internal.Text;
+using Internal.TypeSystem.Ecma;
+
+using Debug = System.Diagnostics.Debug;
+
+namespace ILCompiler.DependencyAnalysis.ReadyToRun
+{
+    public class ManifestAssemblyMvidHeaderNode : ObjectNode, ISymbolDefinitionNode
+    {
+        private ManifestMetadataTableNode _manifestNode;
+
+        public ManifestAssemblyMvidHeaderNode(ManifestMetadataTableNode manifestNode)
+        {
+            _manifestNode = manifestNode;
+        }
+
+        public override ObjectNodeSection Section => ObjectNodeSection.TextSection;
+
+        public override bool IsShareable => false;
+
+        protected internal override int Phase => (int)ObjectNodePhase.Ordered;
+
+        public override int ClassCode => 735231445;
+
+        public override bool StaticDependenciesAreComputed => true;
+
+        public int Offset => 0;
+
+        public int Size => _manifestNode.ManifestAssemblyMvidTableSize;
+
+        public void AppendMangledName(NameMangler nameMangler, Utf8StringBuilder sb)
+        {
+            sb.Append(nameMangler.CompilationUnitPrefix);
+            sb.Append("__ManifestAssemblyMvids");
+        }
+
+        protected override string GetName(NodeFactory nodeFactory)
+        {
+            Utf8StringBuilder sb = new Utf8StringBuilder();
+            AppendMangledName(nodeFactory.NameMangler, sb);
+            return sb.ToString();
+        }
+
+        public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
+        {
+            if (relocsOnly)
+            {
+                return new ObjectData(Array.Empty<byte>(), null, 1, null);
```

Comment:
Nit: Alignment is 1 here, but 0 below?

Reply:
Hehe, thanks for spotting. Actually I believe that 0 is the same as 1 but I changed this line to use 0 too for the sake of consistency in 4th commit.